packages/zmq-android.4.0-8/findlib

```diff
@@ -0,0 +1 @@
+zmq
```

Comment:
There are no findlib files in opam-repository anymore. https://github.com/ocaml/opam-repository/pull/9328